src/sdg/producten/views/product.py

```diff
@@ -251,6 +252,10 @@ def get_context_data(self, **kwargs):
 
         context["languages"] = list(TaalChoices.labels.keys())
         context["product"] = self.product
+        context["doelgroep"] = {
+            "value": self.product.reference_product.most_recent_version.product.generiek_product.doelgroep,
```

Comment:
```suggestion
            "value": self.product.generiek_product.doelgroep,
```

You are making a few unneccessery related lookups.